getting-started/installation.md

```diff
@@ -141,10 +147,32 @@ let config = {
 };
 ```
 
+#### Start MagicMirror² in Server mode on boot
+
+To always start MagicMirror² in server only mode you need to edit the `installers/mm.sh` file otherwise it will start in full mode after a reboot:
+
+1. Backup mm.sh: `cp installers/mm.sh installers/mm.sh.bak`
+2. Edit mm.sh: `nano mm.sh` (you can use your favorite editor)
+3. Replace
+   `DISPLAY=:0 npm start`
+   with
+   `DISPLAY=:0 npm server`
```

Comment:
`npm server` this will not work, must be `npm run server`

Reply:
its running for me with `npm server` changed it to `npm run server` and didn't see any difference.  but i'll admit i don't fully understand how this works, just documenting what i had to do to get it working :)

Reply:
Can also say, that `npm server` does not work for me (or in general as far as i know). Only a few scripts work out of the box, like `start` and `test`

Maybe you got some shortcuts somewhere, but please update here to use "start"